### DISCUSSION TITLE
expire column in mysql data base and delete startegy

### DISCUSSION THREAD

**Discussion by nouhab** [animir/node-rate-limiter-flexible#162] [Q&A] [answered]
hello, 
Thanks you for this great library , 

I would like to understand the usefulness of the expire column, I note that is equal to the date of creation the key + the blockduration . But  
what is its value and how to change it if i do not use the blockduration.

My second question is: I notice that my data gets deleted after a certain period of inactivity for some keys. . what it the starategy ? and can i change it ?:for exemple delete keys that are not used for 24h .

Thank you in advance for the answer.

**Comment by animir** [comment, marked as answer]
@nouhab Hey, thank you.
Sorry for the late response.

`expire` column is necessary to keep track of amount of points consumed during specific period of time.

You can read about [clearExpiredByTimeout](https://github.com/animir/node-rate-limiter-flexible/wiki/Options#clearexpiredbytimeout) on wiki.
I hope, this helps.
